internal/Picker/Picker.module.less

```diff
@@ -32,6 +32,7 @@
 		.item,
 		.itemDecrement,
 		.itemIncrement {
+			overflow: hidden; // FIXME: Overflow text is clipped. But we need a better way.
```

Comment:
Let's deal with this problem on another ticket.

Reply:
> Picker: Check Vertical - Boston AIrport and Airport strings are overlapped

I checked @ybsung 's temporary fix resolves the text overlapping problem, but it looks better to expose the issue to be fixed properly in another ticket rather than a quick fix for component here without discussing about design strategies including a marquee.